lib/node_modules/@stdlib/blas/base/dswap/README.md

```diff
@@ -270,15 +304,6 @@ int main( void ) {
 
 <section class="related">
 
-* * *
```

Comment:
Removing this is fine, but also not necessary for this package. This list of related packages was autogenerated. For the top-level `blas/sswap`, it was removed, as that package had not yet been added to the global namespace.

Reply:
Yes, I thought it will get auto-generated again, as earlier.